test/gtest/uct/ib/test_ib.cc

```diff
@@ -9,11 +9,15 @@
 test_uct_ib::test_uct_ib() : m_e1(NULL), m_e2(NULL) { }
 
 void test_uct_ib::create_connected_entities() {
-    m_e1 = uct_test::create_entity(0);
-    m_e2 = uct_test::create_entity(0);
+    if (m_e1 == NULL) {
```

Comment:
why need to check for NULL?

Reply:
then why need to allocate here?

Reply:
sorry, I meant that there are two variant:
- where we need to allocate entities here
- where they are allocated from the caller's code

Reply:
can we avoid calling create_connected_entities() if they are created from another code?